apigentools/commands/generate.py

```diff
@@ -387,7 +387,17 @@ def pull_repository(self, language, branch=None):
                 )
                 try:
                     self.setup_git_config(cwd=output_dir)
-                    run_command(["git", "merge", self.args.is_ancestor], cwd=output_dir)
+                    run_command(
+                        [
+                            "git",
+                            "merge",
+                            "-Xtheirs",
```

Comment:
```suggestion
```

Reply:
@zippolyte I don't think you will loose any work since there is an explicit merge commit so will still have the original one to verify it.

Reply:
Only if you actually notice that there has been an automatic resolution of your conflict, and did something about it